Gopkg.toml

```diff
@@ -30,6 +30,10 @@ required = [
   name="sigs.k8s.io/controller-tools"
   version="v0.1.9"
 
+[[constraint]]
+  name="k8s.io/klog"
+  revision="aaeac2fedbead93263ae78b80849cac7ff5a5fbb"
```

Comment:
Instead of locking the version here, should we push klog to release a new version and then just update the dep?

Reply:
Already did, it seems they are blocked on a PR and they'll release after it's merged. I'll update once that happens :)